File: CS/Controllers/HomeController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;
using DevExpress.Web.Mvc;
using DevExpress.XtraReports.UI;
using System.Data;
using System.Drawing;
using DevExpress.XtraPrinting;

namespace E4714.Controllers {
    public class HomeController : Controller {
        public ActionResult Index() {
            return View();
        }
    }
}

File: VB/ReportAtRuntimeMvcApp/ReportAtRuntimeMvcApp/Controllers/HomeController.vb
Public Class HomeController
    Inherits System.Web.Mvc.Controller

    Function Index() As ActionResult
        Return View()
    End Function


    Function Designer() As ActionResult
    Dim model As Models.ReportDesignerModel = New Models.ReportDesignerModel()
    Return View(model)
    End Function

    Function Viewer() As ActionResult
        Return View()
    End Function
End Class
